salt/personalised-covers/init.sls

```diff
@@ -100,6 +100,23 @@ personalised-covers-composer-install:
             - personalised-covers-logs
             - personalised-covers-data
 
+personalised-covers-npm-install:
+    cmd.run:
+        - name: npm install
+        - cwd: /srv/personalised-covers
+        - user: {{ pillar.elife.deploy_user.username }}
+        - require:
+            - personalised-covers-repository
+
+personalised-covers-gulp:
+    cmd.run:
+        - name: retry node_modules/.bin/gulp 3
```

Comment:
the retry comes from a crash in dealing with some images, do we need it here?

Reply:
I don't think so.  My attempts to build both locally and in a fresh VM are always successful (no segfaults / OOM).  The journal formula that I copied had a load more Gulp tasks to run in its project, so it will probably be fine here.

Reply:
Ok, if you don't expect failures, only run `node_modules/.bin/gulp` here without the `retry` script.